ncstate-utility-bar.php

```diff
@@ -1,165 +1,195 @@
 <?php
-/**
-* Plugin Name: NC State Utility Bar
-* Plugin URI: https://brand.ncsu.edu
-* Description: Inserts the NC State Brand Utility Bar at the top of every page
-* Version: 0.1
-* Author: NC State University Communications
-* 
-* GitHub Plugin URI: https://github.com/ncstate/ncstate-utility-bar-plugin
-*/
-
-defined( 'ABSPATH' ) or die();
```

Comment:
Is it WordPress practice to not include the "or die()" check even though this file will fatal error when run directly?